captum/insights/api.py

```diff
@@ -47,6 +75,46 @@ def __init__(
         score_func: Optional[Callable] = None,
         use_label_for_attr: bool = True,
     ):
+        r"""
+        Args:
+
+            models (torch.nn.module): PyTorch module (model) for attribution
+                          visualization.
+                          We plan to support visulizing and comparing multiple models
+                          in the future, but currently this supports only a single
+                          model.
+            classes (list of strings): List of strings corresponding to the names of
+                          classes for classification.
+            features (list of BaseFeatures): List of BaseFeatures, which correspond
```

Comment:
should this be list of `BaseFeature` (not plural?) Not sure how our documentation rendering system works. Same with list of strings above.